### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by pedrocamargo**
Brings new style of documentation for examples, where users can download full examples as Python scripts or Jupiter notebooks

Built documentation can be seen here: 
http://aequilibrae.com.s3-website-us-east-1.amazonaws.com/python/dev/index.html

### CODE REVIEW

File: docs/examples/forecasting.py [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+# we build all graphs
+project.network.build_graphs()
+# We get warnings that several fields in the project are filled with NaNs.  Which is true, but we won't use those fields
+
+# %%
+
+# we grab the graph for cars
+graph = project.network.graphs['c']
+
+# let's say we want to minimize free_flow_time
```
Comment (janzill) [inline docs/examples/forecasting.py:50]: ... minimise time, the uncongested value is given in free_flow_time. Splitting hairs :)

**Comment by janzill** [comment]
nice one, I approve or is there more work to be done to transfer all the examples?

File: docs/examples/forecasting.py (hunk 1 above)
Comment (pedrocamargo) [reply docs/examples/forecasting.py:50]: American English here, please!  ;-)  But I agree with you!

**Comment by pedrocamargo** [comment]
@janzill , I am still transferring all the examples, plus I still need to make some examples with the new API for dealing with link types, modes, and links/nodes

**Comment by pedrocamargo** [comment]
I think we now have enough documentation for a release, @janzill 

https://aequilibrae.com/python/dev/_generated/auto_examples/plot_display.html#sphx-glr-generated-auto-examples-plot-display-py

**Comment by pedrocamargo** [comment]
A little XMas present, @jamiecook !!

**Review by janzill — APPROVED** [review]

**Comment by janzill** [comment]
I really like this feature, I've used it quite a bit with matplotlib and scikit-learn

**Comment by pedrocamargo** [comment]
@janzill , Yup. When I figured out how to do it, I just HAD to!!  Thanks!

**Pull request merged**